content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -501,6 +501,37 @@ As this brings Bevy closer to full support of Android, there isn't a need anymor
 
 ![iOS emulator running Bevy](ios%20emulator.png)
 
+## Windows as Entities
+
+<div class="release-feature-authors">authors: @aceeri, @Weibye, @cart</div>
+
+On today's news about the crusade of **X as Entities**, windows are now entities with a [`Window`] component.
+
+You might think this is a bit overkill, but more complex behaviors around windows required extra management of the `WindowId` in other resources or components. As well as led to a lot of cases of confusion where the `WindowDescriptor` resource was effectively read only, but still allowed modification. This hurt usability of windows, since there was a difference between initialising and modifying the window.
+
+Now that [`Window`] is a component, changing the title of a window is the same for both initialising and modifying, no fuss!
+
+```rust
+fn windows_go_brr(mut commands: Commands) {
+    // A new window will be created by any entity with a `Window` component
+    let window = commands.spawn(Window {
+        title: "My window :D".to_owned(),
+        ..default()
+    }).id();
+}
+
+fn modify_windows(mut windows: Query<&mut Window>) {
+    for window in &mut windows {
+        // and modifying fields on the `Window` component will update the window accordingly just the same way!
+        window.title = "My improved window! :D".to_owned();
+    }
+}
+```
+
+This gives us the expressiveness of the ECS while also improving readability/discoverability of how to create, use, and close windows. And any feature we add to entities (relations 🌈, modding support for modifying windows, etc.) will naturally extend to anything that is one.
```

Comment:
"modding support for modifying windows" is unclear to me.

Reply:
Yeah I'm not entirely sure what I was going for here, I'm just gonna keep it simple at just relations.